### ISSUE TITLE
Update Indicator on open screen

### ISSUE THREAD

**Issue by camileevzw** [desktop/desktop#18523]
### The feature request

When opening the Github Desktop application, there is no indication that the app is updating itself, unless one tries to exit the application because they don't see an update indicator, and there is no menu option to start an update..leaving them perplexed, and of course, if the  update is substantial, they didn't have the option to refuse or postpone until a later time, thereby locking them into the update. Now I don't have a problem with updates, that's why I opened the app to check for it, but it should be something that you can check without it locking you up. It's a scheduling issue and a notification issue.

### Proposed solution

Have a visibile indicator or text indicating that an upgrade is available and ask if the user would like to  update the application now or do it later. If they choose to or not to, there should be some kind of text or indicator that an update is starting or is in progress and perhaps, a progress bar with approximate time would be nice.

**Comment by steveward** [comment]
@camileevzw thanks for the issue. We do show a prompt if an update is available directly in the main window of the app, and you can check for updates from the `GitHub Desktop` -> `About GitHub Desktop` menu item (on macOS). This window does show update progress as well. 

We're also tracking the option of opting out of auto-upadtes in https://github.com/desktop/desktop/issues/3410, which seems to align with what you're looking for.

I'm going to close this out, but let me know if you have any further questions or if I'm misunderstanding the feedback here.

**Comment by camileevzw** [comment]
> We do show a prompt if an update is available directly in the main window of the app, and you can check for updates from the `GitHub Desktop` -> `About GitHub Desktop` menu item (on macOS). This window does show update progress as well.

Well, interestingly, after a long time, and while I wrote this report, and then went back to the desktop app, it finally showed that updating aspect, but it took so long that I had time to wait....get irritated and write the report before it showed up...I think is what I'm getting at.  The option to update later would be helpful...but I guess my problem is when you first sign on, maybe the app is being checked and being unresponsive to the point that one thinks it is locked up, before anything shows up.  Just following up on the response...and thanks for pointing it out. Maybe network traffic is to blame, but the thing that got me is that when I tried to leave the app, it warned it was being upgraded, but the window when I went back originally showed nothing...until after I had made the report..and then it shows up. :-(
